prefoil/sampling.py

```diff
@@ -280,6 +280,54 @@ def findSpacing(na):
     return s
 
 
+def tanh_distribution(start, end, n, s0, s1):
```

Comment:
`s0` and `s1` should be keyword arguments for this to work as a spacing function with the `Airfoil` class via `joinedSpacing`.

Reply:
I don't think this is true, but I changed it anyway. Python can unpack arguments even if they are not keyword arguments.